### PR TITLE
Create default SECURITY.md

### PR DESCRIPTION
**Pull request by mbarbero**
Default security policy from Eclipse Foundation's recommendations https://gitlab.eclipse.org/security/best-practices/

### CODE REVIEW

File: SECURITY.md
Comment (pzygielo) [inline SECURITY.md]: As organization hosts many specification projects perhaps referencing Servlet only is not quite right.

Comment (mbarbero) [reply SECURITY.md]: 🤦 Good catch, thanks!

**Comment by mbarbero** [comment]
Any interest in this one? 
/cc @ivargrimstad

File: SECURITY.md
Comment (ivargrimstad) [inline SECURITY.md]: Supported versions are: Check individual repositories for latest versions.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Should have similar text as:

https://github.com/eclipse-ee4j/.github/blob/main/SECURITY.md

**Comment by mbarbero** [comment]
I've proposed a similar, but more accurate wording. Please review. If you're okay with it, I'll suggest a similar change to https://github.com/eclipse-ee4j/.github/blob/main/SECURITY.md.

Thanks!

**Comment by mbarbero** [comment]
Gentle ping. Thanks!

**Review by ivargrimstad — APPROVED** [review]

**Comment by mbarbero** [comment]
I've just added this repository under the scope of the EE4J top-level projects: https://projects.eclipse.org/projects/ee4j/developer. Committers should then be granted access to this repository soon, allowing them to review and merge it. Thanks!

**Comment by mbarbero** [comment]
It took a bit longer than expected because there was a bug in the system that grants permissions on GitHub. You should now be able to merge this one if you're still ok with the content. Thanks!

**Comment by mbarbero** [comment]
Thanks for the merge!

**Pull request merged**
